### DISCUSSION TITLE
How to get involved?

### DISCUSSION THREAD

**Discussion by mlieberman85** [tacosframework/TACOS-spec#1] [General]
Hi, based on the conversation in OpenSSF slack, looking to get a bit more involved. I'm a maintainer on GUAC, steering committee member on SLSA, and want to see how we can potentially collaborate and integrate here.

**Comment by laurenhanford** [comment]
@mlieberman85 Great! Since we're early days, we don't have any recurring discussion times currently. You can drop me an email and I'd be happy to figure out a time to chat.
